proc_setup.py

```diff
@@ -63,7 +65,7 @@
 # dropped, a search-and-replace of "unicode" -> "str" may be done on this
 # module.
 if sys.version_info[0] >= 3:
-    unicode = str
+    str = str
```

Comment:
Maybe also remove this and the comment above now that the search and replace has been done :-)

Reply:
I know, I added that comment myself a while ago ;-) Just reverted the `unicode`/`str` commit so that I could revert the stage-2 fixes without conflict (see top comment for why I wanted that).